library/src/main/java/com/schibsted/spain/barista/BaristaScrollToAction.java

```diff
@@ -0,0 +1,46 @@
+package com.schibsted.spain.barista;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.action.ScrollToAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.v4.widget.NestedScrollView;
+import android.view.View;
+import android.widget.HorizontalScrollView;
+import android.widget.ScrollView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
+import static android.support.test.espresso.matcher.ViewMatchers.withEffectiveVisibility;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anyOf;
+
+public class BaristaScrollToAction implements ViewAction {
```

Comment:
i don't think it's needed.

this action extends the Espresso one, and philosophy of Barista is to make tests easy for devs

if you see two `scrollTo` actions, you will choice Barista one, barista rules!

Also, i don't think is a good idea to explain in the name, because it may chance someday, if android (or support api) changes, should we change the name again?

`NestedScrollToAction` -> `NestedAndXxxxScrollToAction`?